### PR TITLE
Upgrade cloudfriend repo and shortcut runtime to Node10 with support for Node12

### LINKED ISSUE #84: Node.js 8 EOL on Lambda
https://github.com/mapbox/cloudfriend/blob/fdf616a761c91cb93aa7ce1c581faa4418a38f16/lib/shortcuts/lambda.js#L79

We should introduce support for node 12 lambda runtime, and we should also change this default value from `nodejs8.10` to `nodejs10.x`. We should avoid setting the default to `nodejs12.x` because of [this outstanding cloudformation issue](https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/80).

### PR DESCRIPTION
**Pull request by npeternel**
Upgrades shortcut default runtime from Node8 to Node10 in preparation for Node8 EOL.

Closes #84

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-- 8
 - 10
```
Comment (rclark) [inline .travis.yml]: let's add `- 12`

File: test/fixtures/shortcuts/lambda-full.json [hunk 2]
```diff
@@ -103,7 +103,7 @@
             "Arn"
           ]
         },
-        "Runtime": "nodejs6.10",
+        "Runtime": "nodejs10.x",
```
Comment (rclark) [inline test/fixtures/shortcuts/lambda-full.json]: Let's change this test to `nodejs12.x` to prove that the non-default path works.

File: .travis.yml (hunk 1 above)
Comment (npeternel) [reply .travis.yml]: done

File: test/fixtures/shortcuts/lambda-full.json (hunk 2 above)
Comment (npeternel) [reply test/fixtures/shortcuts/lambda-full.json]: done

**Comment by npeternel** [comment]
@rclark Definitely agree with you there

**Pull request merged**